src/main/java/com/ververica/flink/table/gateway/context/ExecutionContext.java

```diff
@@ -125,6 +125,7 @@
 
 	private final Configuration flinkConfig;
 	private final ClusterClientFactory<ClusterID> clusterClientFactory;
+	// For session mode, clusterId is global, while for yarn-per-job mode, clusterId is useless
 	private final ClusterID clusterId;
```

Comment:
`ClusterID` is not needed in `ExecutionContext` now, as `ClusterClientFactory ` can read `ClusterID` from `Configuration` (see `ClusterClientFactory#getClusterId` method). `getClusterId` method can also be removed.